CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ This project adheres to the Node [default version scheme](https://docs.npmjs.com
 
 - Internal: Added ScreenLayout component. This is currently used in the Welcome and Complete screens.
 - Internal: Added user consent screen
+- Internal: Added logo cobranding enterprise feature
```

Comment:
Not sure if there's an agreed convention about this, but for a different enterprise feature PR the wording was such that there is a note that the feature was a premium enterprise feature to assumedly make it a bit clearer that not everyone will have access to it.

Reply:
Nice catch, rewrote the entry in the style hideLogo was